### ISSUE TITLE
output_image 色 異常

### ISSUE THREAD

**Issue by Jingut** [Kazuhito00/Bread-ONNX-Sample#1]
![캡처](https://github.com/Kazuhito00/Bread-ONNX-Sample/assets/58259999/133d7737-f580-4739-9731-5bf0fbe4a9f9)

demo_onnx。pyを実行する時に添付した写真のように出力イメージの色が変に出る理由は何ですか？

**Comment by Kazuhito00** [comment]
![image](https://github.com/Kazuhito00/Bread-ONNX-Sample/assets/37477845/56ffa084-8277-4842-a2c9-bc09efd3a9e9)

私の環境では問題無く動作します。
あなたの試したOS、pipパッケージ、パラメータが分からないと問題を特定できません。

**Comment by Jingut** [comment]
パラメータはdefault値でそのまま進め、OSはwindow10でanacondaを利用して進めました。
また、pip listの場合、以下のようにインストールされています。

Package                      Version
---------------------------- ---------
absl-py                      1.4.0
astunparse                   1.6.3
cachetools                   5.3.1
certifi                      2023.7.22
charset-normalizer           3.2.0
click                        8.1.7
colorama                     0.4.6
coloredlogs                  15.0.1
cycler                       0.11.0
ffmpeg                       1.4
Flask                        2.2.5
flatbuffers                  2.0.7
gast                         0.4.0
google-auth                  2.22.0
google-auth-oauthlib         0.4.6
google-pasta                 0.2.0
grpcio                       1.57.0
h5py                         3.8.0
humanfriendly                10.0
idna                         3.4
imageio                      2.31.2
importlib-metadata           6.7.0
imutils                      0.5.4
itsdangerous                 2.1.2
Jinja2                       3.1.3
joblib                       1.3.2
keras                        2.7.0
Keras-Preprocessing          1.1.2
kiwisolver                   1.4.5
kornia                       0.6.12
libclang                     16.0.6
Markdown                     3.4.4
MarkupSafe                   2.1.3
matplotlib                   3.4.2
mpmath                       1.3.0
networkx                     2.6.3
numpy                        1.21.6
oauthlib                     3.2.2
onnxruntime                  1.14.1
opencv-python                4.5.4.60
opt-einsum                   3.3.0
packaging                    23.2
Pillow                       9.5.0
pip                          23.2.1
protobuf                     3.20.3
pyasn1                       0.5.0
pyasn1-modules               0.3.0
pyparsing                    3.1.1
pyreadline                   2.1
python-dateutil              2.8.2
pytictoc                     1.5.3
pytorch-msssim               1.0.0
PyWavelets                   1.3.0
requests                     2.31.0
requests-oauthlib            1.3.1
rsa                          4.9
scikit-image                 0.19.3
scikit-learn                 1.0.1
scipy                        1.7.3
setuptools                   68.1.2
six                          1.16.0
sympy                        1.10.1
tensorboard                  2.11.2
tensorboard-data-server      0.6.1
tensorboard-plugin-wit       1.8.1
tensorflow                   2.7.0
tensorflow-estimator         2.7.0
tensorflow-io-gcs-filesystem 0.31.0
termcolor                    2.3.0
threadpoolctl                3.1.0
tifffile                     2021.11.2
torch                        1.13.1
torchvision                  0.14.1
tqdm                         4.62.0
typing_extensions            4.7.1
urllib3                      1.26.16
utils                        1.0.1
Werkzeug                     2.2.3
wheel                        0.41.2
wrapt                        1.15.0
zipp                         3.15.0

**Comment by Kazuhito00** [comment]
追加のコメントをいただいたと思っていましたが、既に解決されましたか？
もしまだ問題がある場合は、再度Issueを起票してください。

**Comment by Jingut** [comment]
個人的に問題を解決しました。
私が解決した方法はRepo ownerが提供してくださったcvuiをインポートせず、別途にopencvのtrackbarposライブラリを利用して実行するので問題が解決されました。 
私の Window 10環境ではcvui。pyがうまく作動しない理由を見つけることができませんでした。
